.travis.yml

```diff
@@ -51,7 +52,7 @@ jobs:
       - export PATHSAVE=$PATH
       - export PATH=$PWD/linux/bin:$PATH
       - export LD_LIBRARY_PATH=$PWD/linux/lib:$LD_LIBRARY_PATH
-      - PROTEUS_OPT="-w -O2 -UNDEBUG" FC=gfortran CC=mpicc CXX=mpicxx make develop N=2
+      - PROTEUS_OPT="-w -O2 -UNDEBUG" FC=gfortran CC=mpicc CXX=mpicxx make develop N=1
```

Comment:
This avoids timeout on travis due to long compilation. My guess is that the worker is going out of memory because of many template instantiations and swaps a lot. This an be enabled again when we moved to the dict of pyarray objects and reduce the size of the API.